liiweb/templates/liiweb/layouts/list.html

```diff
@@ -0,0 +1,39 @@
+{% extends "peachjam/layouts/main.html" %}
+
+{% block page-content %}
+  <section class="pt-5 pb-5">
+    <div class="container">
+      {% block grouped-links %}
+        {% if groups %}
+          {% for group in groups %}
+            <h4 class="heading-underlined">
```

Comment:
Two thoughts on this:

1. this template is only used by one other template -- it seems premature to make it re-usable (especially since line 21 refers to courts)
2. please put the court class groupings into three columns: so "Magistrate Court" is one column, "Supreme Court" the next etc. eg. `col-md-3` should work, even if it wraps between rows.

Reply:
refactored. Put the list.html logic in judgement_list.html (obviously without the block logic) and deleted list.html